### DISCUSSION TITLE
Private Comment

### DISCUSSION THREAD

**Discussion by thumDer** [outline/outline#6582] [Ideas]
An idea came up today, which might seem quite unusual, we would have valid usecases for. What if a comment could be made private, and only the tagged users, and the author could see it? Obviously, the whole thread would inherit the visibility, and might be extended in subsequent comments, if other users are tagged.

**Comment by tommoor** [comment]
It's an interesting idea, i haven't seen anything like this in other tools. 

I think in the short term it's probably best to keep private messages distinctly separated.

**Reply by thumDer** [reply]
But I think the most important in comments is the context. Using private messages on other platforms is disconnected. There is a huge value in linking the message to a section of a page.
I haven't seen anything like either, but this shouldn't mean that outline can't be a pioneer! ;)
